compiler-rt/test/sanitizer_common/TestCases/Darwin/malloc_zone.cpp

```diff
@@ -17,6 +17,8 @@
 // UBSan does not install a malloc zone.
 // XFAIL: ubsan
 //
+// Curently fails on darwin/lsan rdar://145873843
```

Comment:
Radar links are appropriate for commit messages, but the community no longer accepts them in comments.

Reply:
This will need to be fixed in main and backported along with this patch.